packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -739,14 +739,14 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 	 * be based off of local ids.
 	 * @returns The routes of blobs that were deleted.
 	 */
-	public deleteSweepReadyNodes(sweepReadyBlobRoutes: string[]): string[] {
+	public deleteSweepReadyNodes(sweepReadyBlobRoutes: readonly string[]): readonly string[] {
 		// If sweep for attachment blobs is not enabled, return empty list indicating nothing is deleted.
 		if (this.mc.config.getBoolean(disableAttachmentBlobSweepKey) === true) {
 			return [];
 		}
 
 		this.deleteBlobsFromRedirectTable(sweepReadyBlobRoutes);
-		return Array.from(sweepReadyBlobRoutes);
+		return sweepReadyBlobRoutes;
```

Comment:
You 100% confident about this?  Worth doing in a separate PR just to be safe?

Reply:
This array is not changed so this is safe. But maybe worth doing in a separate PR along with some other optmizations like using set everywhere instead of arrays.